contracts/src/periphery/EigenDACertVerifierV3.sol

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifier} from "src/interfaces/IEigenDACertVerifier.sol";
+import {IEigenDAThresholdRegistry} from "src/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDABatchMetadataStorage} from "src/interfaces/IEigenDABatchMetadataStorage.sol";
+import {IEigenDASignatureVerifier} from "src/interfaces/IEigenDASignatureVerifier.sol";
+import {OperatorStateRetriever} from "lib/eigenlayer-middleware/src/OperatorStateRetriever.sol";
+import {IRegistryCoordinator} from "lib/eigenlayer-middleware/src/interfaces/IRegistryCoordinator.sol";
+import {EigenDACertVerificationV3Lib as CertV3Lib} from "src/libraries/V3/EigenDACertVerificationV3Lib.sol";
+import {EigenDAV3Cert} from "src/libraries/V3/EigenDATypesV3.sol";
+import {EigenDATypesV2 as DATypesV2} from "src/libraries/V2/EigenDATypesV2.sol";
+import {EigenDATypesV1 as DATypesV1} from "src/libraries/V1/EigenDATypesV1.sol";
+import {EigenDACertVerificationV2Lib as CertV2Lib} from "src/libraries/V2/EigenDACertVerificationV2Lib.sol";
+
+contract EigenDACertVerifierV3 is IEigenDACertVerifier {
+    error InvalidSecurityThresholds();
+
+    IEigenDAThresholdRegistry public immutable eigenDAThresholdRegistry;
+
+    IEigenDASignatureVerifier public immutable eigenDASignatureVerifier;
+
+    IRegistryCoordinator public immutable registryCoordinator;
+
+    DATypesV1.SecurityThresholds public securityThresholds;
+
+    bytes public quorumNumbersRequired;
+
+    uint8 public constant CERT_VERSION = 3;
+
+    constructor(
+        IEigenDAThresholdRegistry _eigenDAThresholdRegistry,
```

Comment:
Can we feasibly rename this contract to `BlobParamsRegistry`? Would it cause bad breaking changes? Ive asked this many times to arch but he was too lazy. It no longer contains thresholds (they are in this contract) so the name makes 0 sense.

Reply:
Yes, but I would argue that it would be a separate change in a separate PR. Naming changes are fine in my book.

Reply:
sg. created https://linear.app/eigenlabs/issue/EGDA-1356/rename-thresholdregistry-blobparamsregistry. feel free to resolve this thread